resources/definitions/m180.def.json

```diff
@@ -36,6 +35,21 @@
                 [ 18, 35 ]
             ]
         },
+        "machine_max_feedrate_z": {
+            "default_value": 400
+        },
+        "steps_per_mm_x": {
```

Comment:
Shouldn't this be machine_steps_per_mm_x ? And the same for the rest of these settings...
It's merged, it can be changed in master now :-)

Reply:
Nice catch! Fixed in a477b8cdc

Reply:
Nice fix ^^